lib/hooks/useHomepage.js

```diff
@@ -2,7 +2,7 @@ import useSWR from 'swr';
 import fetcher from './fetcher';
 export default function useHomepage(siteId) {
   if (siteId) {
-    const { data, error } = useSWR(`/api?site=${siteId}`, fetcher);
+    const { data, error } = useSWR(`/api/groups`, fetcher);
```

Comment:
Previously the api just went to `/api` when it was just fetching groups. I organized that to fetch the correct group route instead of the index route